### PR TITLE
update static pages

### PR DESCRIPTION
**Pull request by vinhnd2010**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,45 +1,36 @@
...
-  # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
-
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-end
-
+source 'https://rubygems.org'
+
+gem 'rails',        '4.2.3'
```
Comment (dieunb) [inline Gemfile]: replace `'` with `"`

File: app/assets/javascripts/static_pages.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title='')
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: remove `()` and `page_title = ""`

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = "Framgia E-learning System"
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: use i18n for this String

File: app/helpers/static_pages_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: do you use this file? if no, please remove it.

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<footer class = "footer">
+	<small>
```
Comment (dieunb) [inline app/views/layouts/_footer.html.erb]: indent

File: app/views/static_pages/about.html.erb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, 'About') %>
```
Comment (dieunb) [inline app/views/static_pages/about.html.erb]: remove `()`

File: app/views/static_pages/help.html.erb [hunk 9]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, "Help") %>
```
Comment (dieunb) [inline app/views/static_pages/help.html.erb]: remove `()`

File: app/views/static_pages/home.html.erb [hunk 10]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "Home") %>
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: remove `()`

File: config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  
+  root 'static_pages#home'
```
Comment (dieunb) [inline config/routes.rb]: replace `'` with `"`

**Comment by hellovietnam93** [comment]
Ok

File: app/views/layouts/application.html.erb [hunk 12]
```diff
@@ -1,14 +1,19 @@
...
+  <title><%= full_title(yield(:title)) %></title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
-
+  <%= render 'layouts/header' %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: replace `'` with `"` and apply for all your code

File: app/views/layouts/_footer.html.erb [hunk 13]
```diff
@@ -0,0 +1,13 @@
+<footer class = "footer">
+  <small>
+    The <a href="">Framgia E-learning System</a>
+    by <a href="">Traninng team</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (dieunb) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/layouts/_footer.html.erb [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+      <li><%= link_to "Contact", "#" %></li>
```
Comment (dieunb) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 15]
```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "FELS", root_path, id: "logo" %>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 16]
```diff
@@ -0,0 +1,13 @@
...
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   root_path %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+        <li><%= link_to "Help",   help_path %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+        <li><%= link_to "Login", "#" %></li>
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: i18n

File: app/helpers/application_helper.rb [hunk 19]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title=""
+    base_title = t("FELS")
+    if page_title.empty?
+      base_title
+    else
+      page_title+" | "+base_title
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: `page_title + " | " + base_title`

File: app/helpers/application_helper.rb [hunk 20]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title=""
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: `page_title = ""`

**Pull request merged**